src/main.rs

```diff
@@ -2441,15 +2463,28 @@ mod tests {
             aver_eq!(
                 tb,
                 guest
-                    .ssh_command_l2("cat /proc/cmdline | grep -c 'VFIOTAG'")
+                    .ssh_command_l2_1("cat /proc/cmdline | grep -c 'VFIOTAG'")
```

Comment:
`grep -c VFIOTAG /proc/cmdline`

Reply:
Let's see if the CI passes, I'll update right after ;)